python/hackathon_app/server/main.py

```diff
@@ -60,7 +60,7 @@ def get_hackathons():
         cred_file=app.config["GOOGLE_APPLICATION_CREDENTIALS"],
     )
     try:
-        hackathons = [h.name for h in sheets_client.get_hackathons()]
+        hackathons = [h.label for h in sheets_client.get_hackathons()]
```

Comment:
like we discussed:

```suggestion
        hackathons = {h.name: h.label for h in sheets_client.get_hackathons()}
```

also, give updating the front-end a shot: might just be a one-line change here https://github.com/looker-open-source/sdk-examples/blob/master/python/hackathon_app/frontend/src/RegisterScene.tsx#L107